packages/editor/src/components/rich-text/index.js

```diff
@@ -234,7 +234,13 @@ export class RichText extends Component {
 	}
 
 	createRecord() {
-		const range = window.getSelection().getRangeAt( 0 );
+		const selection = window.getSelection();
+
+		if ( selection.rangeCount === 0 ) {
+			return null;
+		}
+
+		const range = selection.getRangeAt( 0 );
```

Comment:
It might be better to still create the value, but not pass a range in this case.